tests/admin_default_site/tests.py

```diff
@@ -1,6 +1,7 @@
 from django.contrib import admin
 from django.contrib.admin import sites
 from django.test import SimpleTestCase, override_settings
+from .sites import CustomAdminSite
```

Comment:
Blank line is missing:
```suggestion

from .sites import CustomAdminSite
```